projects/smoke_pca9539r/src/main.c

```diff
@@ -0,0 +1,123 @@
+/* Smoke test for pca9539r gpio expander. i2c and other settings based off of
+   test_pca9539r_gpio_expander.c Program sets all pins as inputs, and low state, then outputs with
+   high state, and checks configuration registers against expected results Then toggles gpio state
+   on all pins every second */
+
+#include "gpio.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "log.h"
+#include "pca9539r_gpio_expander.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
```

Comment:
`test_helpers.h` shouldn't be included outside of a test.

Reply:
(outside of unit tests)